node-notes-cli/app.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+
+const arg1 = process.argv[2];
+
+if (arg1 === 'read') {
+  fs.readFile('./data.json', 'utf8', (err, data) => {
+    if (err) {
+      throw err;
+    }
+    console.log(data);
```

Comment:
For the `read` option you should only output each note, with each note on its own line